pydatastructs/linear_data_structures/linked_lists.py

```diff
@@ -508,3 +512,219 @@ def __getitem__(self, index):
             current_node = current_node.next
             counter += 1
         return current_node
+
+class SinglyCircularLinkedList(SinglyLinkedList):
+    """
+    Represents Singly Circular Linked List.
+
+    Parent Class
+    ============
+
+    SinglyLinkedList.
+
+    Examples
+    ========
+
+    >>> from pydatastructs import SinglyCircularLinkedList
+    >>> scll = SinglyCircularLinkedList()
+    >>> scll.append(6)
+    >>> scll[0].data
+    6
+    >>> scll.head.data
+    6
+    >>> scll.append(5)
+    >>> scll.append_left(2)
+    >>> print(scll)
+    [2, 6, 5]
+    >>> scll[0].data = 7.2
+    >>> scll.extract(1).data
+    6
+    >>> print(scll)
+    [7.2, 5]
+
+    References
+    ==========
+
+    .. [1] https://en.wikipedia.org/wiki/Linked_list#Circular_linked_list
+    .. [2] https://www.geeksforgeeks.org/circular-singly-linked-list-insertion/
```

Comment:
```suggestion
```

Reply:
The wiki page does not have enough info so I added other references.

Reply:
The info on the Wiki is not enough. Should I add a reference from other sites or leave it as it is?

Reply:
You can add links to university lectures.

Reply:
Update Wiki References.